src/material-experimental/mdc-menu/menu.scss

```diff
@@ -19,17 +19,21 @@ mat-menu {
 }
 
 .mat-mdc-menu-panel {
+  @include a11y.high-contrast(active, off) {
+    outline: solid 1px;
+  }
+}
+
+// We need to increase the specificity for these styles to ensure we are not overriden by MDC's
+// .mdc-menu-surface styles. This can happen if the MDC styles are loaded in after our styles.
+.mat-mdc-menu-panel.mat-mdc-menu-panel {
```

Comment:
In what case would MDC's menu styles come in after ours? We should have control over the load order in this case.

Reply:
I have already run into a case internally where this happens, but I'm not sure how they managed to cause it

Reply:
Is it conflicting with our styles or MDC's? It might be the same issue I was trying to solve with https://github.com/angular/components/pull/22562. I think this is fine for now, but we should just understand why it's happening.

Reply:
I'm not certain, but if that were the case I should be able to reproduce the issue on our devapp, but I cannot

For context, the app which was able to hit this edge case had a lot of duplicate css (I counted 12 `<styles>` divs with MDC's menu styles and 6 with our styles). I assumed it was something with the way they were importing styles